src/workspacer.Native/Native/WindowsManager.cs

```diff
@@ -30,10 +30,13 @@ public class WindowsManager : IWindowsManager
 
         private Dictionary<WindowsWindow, bool> _floating;
 
-        public event WindowCreateDelegate WindowCreated;
-        public event WindowDelegate WindowDestroyed;
-        public event WindowUpdateDelegate WindowUpdated;
+        public event WindowAddDelegate AddWindowTrigger;
+        public event WindowDelegate RemoveWindowTrigger;
+        public event WindowUpdateDelegate UpdateWindowTrigger;
+
         public event WindowFocusDelegate WindowFocused;
+        public event WindowDelegate WindowUpdated;
+        public event WindowDelegate WindowClosed;
```

Comment:
This may be a stupid question, but technically speaking... What is the difference between `WindowClosed` and `WindowDestroyed`? Might be worth a code-doc?

Reply:
Code doc for all of the new window events will be added in next day or 2 to clarify the use and origin